server/api/src/rsGeneralAdmin.cpp

```diff
@@ -677,7 +677,8 @@ _rsGeneralAdmin( rsComm_t *rsComm, generalAdminInp_t *generalAdminInp ) {
             return irods::create_user(*rsComm,
                                       generalAdminInp->arg2 ? generalAdminInp->arg2 : "",
                                       generalAdminInp->arg3 ? generalAdminInp->arg3 : "",
-                                      generalAdminInp->arg5 ? generalAdminInp->arg5 : "");
+                                      generalAdminInp->arg5 ? generalAdminInp->arg5 : "",
+                                      generalAdminInp->arg4 ? generalAdminInp->arg4 : "");
```

Comment:
just caught my eye that these are 'out of order'... disregard if this is confirmed to be correct.

Reply:
Yeah, the order from iadmin is:
 1. action ("add")
 2. name
 3. type
 4. zone name
 5. auth string

I can change the order if desired. It's a server-only interface so it can be changed freely in the future